users/postgres/users.go

```diff
@@ -31,27 +35,54 @@ type userRepository struct {
 
 // New instantiates a PostgreSQL implementation of user
 // repository.
-func New(db Database) users.UserRepository {
+func NewUserRepo(db Database) users.UserRepository {
```

Comment:
There is a reason to change this name?

Reply:
Yes, because we have group repo also, similar as in things service we have channel repo and things repo